README.md

```diff
@@ -29,6 +28,8 @@ Maintainer: @Lapineige
 
 ## Installation information
 
+During the installation you will be prompted if you want to set this instance as public, which means any people (not just Yunohost users) can consult you Shaarli.
```

Comment:
During the installation app will prompt you to install itself as public or private, if you choose public then anyone (not just Yunohost users) would be able to access your Shaarli instance.